android/build.gradle

```
@@ -47,8 +47,8 @@ android {
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', DEFAULT_MIN_SDK_VERSION)
         targetSdkVersion safeExtGet('targetSdkVersion', DEFAULT_TARGET_SDK_VERSION)
-        versionCode 1
-        versionName "1.0"
+        versionCode 2
+        versionName "1.1.1"
```

```suggestion
        versionCode 1
        versionName "1.0"
```

Please revert this as versioning is handled on the master branch directly.